src/Bicep.Core/FileSystem/PathHelper.cs

```diff
@@ -185,7 +185,7 @@ public static bool HasExtension(Uri uri, string extension)
 
         public static bool HasBicepExtension(Uri uri) => HasExtension(uri, BicepExtension);
 
-        public static bool HasBicepparamsExension(Uri uri) => HasExtension(uri, BicepParamsExtension);
```

Comment:
Should this be `HasBicepparamExtension` (since the extension is `.bicepparam`, not `.biceparams`

Reply:
Sure, done.

Lots of usages of "BicepParametersFile" and "BicepParamsFile".  I could change those, but not sure we want to since they match the doc usage: https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/parameter-files?tabs=Bicep